src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagResetSeverity.java

```diff
@@ -0,0 +1,42 @@
+/*
+WriteTag
+tag = @author
+tagFormat = Mohanad
+tagSeverity = warning
```

Comment:
please remove this to avoid any confusion that this Input is using config from Input file.
It is exceptional Input, it should looks like exceptional.

Reply:
done